spec/factories/capital_summary_factory.rb

```diff
@@ -0,0 +1,31 @@
+FactoryBot.define do
+  factory :capital_summary do
+    assessment
+
+    trait :pending do
+      capital_assessment_result { 'pending' }
+    end
+
+    trait 'summarised' do
+      capital_assessment_result { 'summarised' }
+      lower_threshold { 3_000 }
+      upper_threshold { 8_000 }
```

Comment:
The thresholds are configurable - so likely to change and therefore should not be hard coded here. So this should be:
```ruby
 lower_threshold { Threshold.value_for(:capital_lower, at: Time.now) }
 upper_threshold { Threshold.value_for(:capital_upper, at: Time.now) }
```

Reply:
In a factory?  We don't want all the tests failing after a certain date just because the thresholds have changed.

Reply:
If a change of threshold causes this to break that needs to occur in the tests. With hard coded thresholds that will not happen.